app/src/androidTest/java/com/github/fribourgsdp/radio/LyricsGetterTest.kt

```diff
@@ -102,22 +106,31 @@ class LyricsGetterTest {
"Music\"}},{\"music_genre\":{\"music_genre_id\":21,\"music_genre_parent_id\":34,\"music_genre_name\":\"Rock\",\"music_genre_name_extended\":\"Rock\",\"music_genre_vanity\":\"Rock\"}}]}}},{\"track\":{\"track_id\":169230457,\"track_name\":\"Hurricane (Live at Montreal Forum, Montreal, Quebec - December 1975)\",\"track_name_translation_list\":[],\"track_rating\":24,\"commontrack_id\":95050462,\"instrumental\":0,\"explicit\":1,\"has_lyrics\":1,\"has_subtitles\":0,\"has_richsync\":0,\"num_favourite\":0,\"album_id\":32252440,\"album_name\":\"Travelin' Thru, 1967 - 1969: The Bootleg Series, Vol. 15\",\"artist_id\":13,\"artist_name\":\"Bob Dylan\",\"track_share_url\":\"https:\\/\\/www.musixmatch.com\\/lyrics\\/Bob-Dylan\\/Hurricane-Live-at-Montreal-Forum-Montreal-Quebec-December-1975?utm_source=application&utm_campaign=api&utm_medium=FribourgSDP%3A1409622437882\",\"track_edit_url\":\"https:\\/\\/www.musixmatch.com\\/lyrics\\/Bob-Dylan\\/Hurricane-Live-at-Montreal-Forum-Montreal-Quebec-December-1975\\/edit?utm_source=application&utm_campaign=api&utm_medium=FribourgSDP%3A1409622437882\",\"restricted\":0,\"updated_time\":\"2019-05-01T03:22:05Z\",\"primary_genres\":{\"music_genre_list\":[{\"music_genre\":{\"music_genre_id\":34,\"music_genre_parent_id\":0,\"music_genre_name\":\"Music\",\"music_genre_name_extended\":\"Music\",\"music_genre_vanity\":\"Music\"}},{\"music_genre\":{\"music_genre_id\":21,\"music_genre_parent_id\":34,\"music_genre_name\":\"Rock\",\"music_genre_name_extended\":\"Rock\",\"music_genre_vanity\":\"Rock\"}}]}}},{\"track\":{\"track_id\":86724560,\"track_name\":\"Hurricane - Live at Memorial Auditorium, Worcester, MA - November 1975\",\"track_name_translation_list\":[],\"track_rating\":24,\"commontrack_id\":14399029,\"instrumental\":0,\"explicit\":1,\"has_lyrics\":1,\"has_subtitles\":1,\"has_richsync\":0,\"num_favourite\":0,\"album_id\":21057711,\"album_name\":\"The Bootleg Series, Vol. 5 - Bob Dylan Live 1975: The Rolling Thunder Revue\",\"artist_id\":13,\"artist_name\":\"Bob Dylan\",\"track_share_url\":\"https:\\/\\/www.musixmatch.com\\/lyrics\\/Bob-Dylan\\/Hurricane-Live?utm_source=application&utm_campaign=api&utm_medium=FribourgSDP%3A1409622437882\",\"track_edit_url\":\"https:\\/\\/www.musixmatch.com\\/lyrics\\/Bob-Dylan\\/Hurricane-Live\\/edit?utm_source=application&utm_campaign=api&utm_medium=FribourgSDP%3A1409622437882\",\"restricted\":0,\"updated_time\":\"2013-11-28T18:04:50Z\",\"primary_genres\":{\"music_genre_list\":[{\"music_genre\":{\"music_genre_id\":14,\"music_genre_parent_id\":34,\"music_genre_name\":\"Pop\",\"music_genre_name_extended\":\"Pop\",\"music_genre_vanity\":\"Pop\"}},{\"music_genre\":{\"music_genre_id\":21,\"music_genre_parent_id\":34,\"music_genre_name\":\"Rock\",\"music_genre_name_extended\":\"Rock\",\"music_genre_vanity\":\"Rock\"}},{\"music_genre\":{\"music_genre_id\":34,\"music_genre_parent_id\":0,\"music_genre_name\":\"Music\",\"music_genre_name_extended\":\"Music\",\"music_genre_vanity\":\"Music\"}}]}}}]}}}"
```

Comment:
Could you just explain what is the purpose of these huge lines?

Reply:
This is just to _mock_ a response from musixmatch, to see if it is well interpreted. This was already there in my original pull request.
Here, it is indeed the response musixmatch gives you when you ask it to search for a song named "Hurricane" by "Bob Dylan".